Dockerfile

```diff
@@ -6,8 +6,12 @@ COPY . $HOME/application
 
 WORKDIR $HOME/application/wellregistry
 
-RUN pip install --no-cache-dir -r ../requirements.txt
-RUN pip install --no-cache-dir -r ../requirements-prod.txt
+RUN apt-get update \
+ && apt-get install gcc libpq-dev python3-dev -y \
+ && pip install --no-cache-dir -r ../requirements-prod.txt \
+ && apt-get autoremove --purge -y gcc libpq-dev python3-dev \
+ && rm -rf /var/lib/apt/lists/* \
+ && pip install --no-cache-dir -r ../requirements.txt
```

Comment:
The gcc install over build-essential reduces the size by ~ 93mb
The autoremove/purge reduced the size by ~232mb
The deletion of apt-get update files reduces the size by ~21mb

Image size went from a bloated 614mb down to 258mb.

Reply:
It looks like the image was originally 212mb and that the increased size for the new python modules is a reasonable 46mb. We added quite a few modules since the ZFR.